package.json

```diff
@@ -20,24 +20,24 @@
     "vue": "^2.6.10",
     "vue-ctk-date-time-picker": "^2.1.1",
     "vue-notification": "^1.3.16",
-    "vue-router": "^3.0.3",
+    "vue-router": "^3.0.7",
     "vue-tags": "^1.0.4",
     "vuejs-auto-complete": "^0.9.0",
-    "vuex": "^3.0.1",
+    "vuex": "^3.1.1",
     "xmlhttprequest": "^1.8.0"
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "^3.9.2",
     "@vue/cli-plugin-eslint": "^3.9.2",
     "@vue/cli-plugin-unit-jest": "^3.9.0",
-    "@vue/cli-service": "^3.9.2",
+    "@vue/cli-service": "^3.9.3",
     "@vue/test-utils": "1.0.0-beta.29",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "^10.0.2",
     "babel-jest": "^23.6.0",
     "csv-parse": "^4.4.3",
     "dom-parser": "^0.1.6",
-    "eslint": "^5.16.0",
+    "eslint": "^5.1.0",
```

Comment:
I think the current version is actually `6.1.0`, not `5.1.0`.

Reply:
When I tried to update eslint to 6.1.0, linting subsequently failed with the following message:
` Error: Failed to load plugin 'vue' declared in 'package.json': Cannot find module 'eslint/lib/util/traverser'`

Reply:
But you do realize that `5.1.0` is technically older than `5.16.0` which is what was previously used right?

Reply:
It would make more sense to stay with 5.16.0 rather than revert several versions to 5.1.0.

Reply:
Looking at the change, the reversion is likely a mistake after previously trying 6.1.0 and naively switching back to 5.1.0. I will return it to 5.16.0.